packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/data.ts

```diff
@@ -554,107 +554,107 @@ export function getData() {
             name: 'latest',
             results: {
                 'integrated charts large scale benchmark initial load': {
-                    timeMs: 674.5566669999971,
-                    memoryUsage: 264565736,
-                    heapUsed: 197365736,
+                    timeMs: 603.7953330000018,
+                    memoryUsage: 255888768,
+                    heapUsed: 188688768,
                     canvasCount: 35,
                     canvasBytes: 67200000,
                 },
                 'integrated charts large scale benchmark after load 1x legend toggle': {
-                    timeMs: 854.0436669999981,
-                    memoryUsage: 268692344,
-                    heapUsed: 201492344,
+                    timeMs: 838.5170829999988,
+                    memoryUsage: 259868896,
+                    heapUsed: 192668896,
                     canvasCount: 35,
                     canvasBytes: 67200000,
                 },
                 'integrated charts large scale benchmark after load 4x legend toggle': {
-                    timeMs: 1581.8064999999988,
-                    memoryUsage: 269178424,
-                    heapUsed: 201978424,
+                    timeMs: 1590.578999999998,
+                    memoryUsage: 260344176,
+                    heapUsed: 193144176,
                     canvasCount: 35,
                     canvasBytes: 67200000,
                 },
                 'large-dataset benchmark initial load': {
-                    timeMs: 5530.321125,
-                    memoryUsage: 787808480,
-                    heapUsed: 772448480,
+                    timeMs: 6658.801583000001,
```

Comment:
That's a pretty big increase

Reply:
Agreed - we're probably going to be looking at ways to optimise the number of change-detections needed in the future, specifically either:
- optimising the changing of properties in the first place, so the extra overhead here becomes less relevant.
- batching rendering, which will mean for this specific use case we'd only be performing O(1) update per scene-graph property rather than O(N).